keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -235,17 +235,11 @@ def test_saved_model(self, save_format, filename):
 @pytest.mark.large
 class RetinaNetSmokeTest(tf.test.TestCase):
     def test_backbone_preset(self):
-        weights = [
-            "csp_darknet_tiny",
-            "efficientnetv2_s",
-            "efficientnetv2_b0",
-            "mobilenet_v3_small",
-            "resnet50",
-            "resnet50_v2",
-        ]
-        for weight in weights:
+        for preset in keras_cv.models.RetinaNet.presets:
+            if preset in keras_cv.models.RetinaNet.presets_with_weights:
+                continue
             model = keras_cv.models.RetinaNet.from_preset(
```

Comment:
If I understand correctly, this is what Jonathan was suggesting. This is fine by me.

Reply:
Yes, thank you!